### ISSUE TITLE
[Bug]: Valet\Herd::getBasePhpPort(): Return value must be of type int, string returned

### ISSUE THREAD

**Issue by ericmp33** [beyondcode/herd-community#513]
### Operating system version

Windows 11

### System architecture

Intel (x86)

### Herd Version

1.0.0

### PHP Version

PHP 8.3.4 (cli) (built: Mar 13 2024 11:42:47) (NTS Visual C++ 2019 x64)

### Bug description

I did:
```ps
herd restart
```
I got:
```php
PHP Fatal error:  Uncaught TypeError: Valet\Herd::getBasePhpPort(): Return value must be of type int, string returned in phar://C:/Users/eric/.config/herd/bin/herd.phar/cli/Valet/Herd.php:30
```

### Steps to reproduce

1. Install Herd (fresh)
2. Run `herd restart`

**Comment by bennajah** [comment]
same issue on windows

**Comment by mpociot** [comment]
Fixed in version 1.0.1 
https://herd.laravel.com/changelog/windows#101
